siege_utilities/geo/providers/redistricting_data_hub.py

```
@@ -0,0 +1,1084 @@
+            "schwartzberg": schwartzberg(geom),
+        })
+
+    return pd.DataFrame(records)
+
+
+# ---------------------------------------------------------------------------
+# Plan comparison
+# ---------------------------------------------------------------------------
+
+def compare_plans(
+    plan_a: "gpd.GeoDataFrame",
```

Ruff (0.15.11)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)